src/Functional.CQS.AOP.CommonTestInfrastructure/DummyObjects/DummyAsyncCommandHandlerThatFails.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects
+{
+	/// <summary>
+	/// Sample <see cref="IAsyncCommandHandler{TCommand, TError}"/> implementation.  Returns a failure result.
+	/// </summary>
+	public class DummyAsyncCommandHandlerThatFails : IAsyncCommandHandler<DummyAsyncCommandThatFails, DummyAsyncCommandError>
+	{
+		private static readonly DummyAsyncCommandThatFails _command = new DummyAsyncCommandThatFails();
```

Comment:
Looks like deleting this instance of the unused `_command` was missed.

Reply:
Done